py4DSTEM/preprocess/preprocess.py

```diff
@@ -281,6 +281,12 @@ def bin_data_diffraction(
     # set calibration pixel size
     datacube.calibration.set_Q_pixel_size(Qpixsize)
 
+    # remake Cartesian coordinate system
+    datacube._qyy,datacube._qxx = np.meshgrid(
+        np.arange(0,datacube.Q_Ny),
+        np.arange(0,datacube.Q_Nx)
+    )
+
```

Comment:
the `._qxx,._qyy` properies are set in datacube.calibrate - we shouldn't overwrite them here